src/main/java/uk/gov/hmcts/reform/civil/service/SendSDOBulkPrintService.java

```diff
@@ -32,7 +33,7 @@ public void sendSDOToDefendantLIP(CaseData caseData) {
                 String documentId = documentUrl.substring(documentUrl.lastIndexOf("/") + 1);
                 byte[] letterContent;
                 try {
-                    letterContent = documentDownloadService.downloadDocument(documentId).file().getInputStream().readAllBytes();
+                    letterContent = documentDownloadService.downloadDocument(BEARER_TOKEN, documentId).file().getInputStream().readAllBytes();
```

Comment:
This seems to be hard coded auth token here.

Reply:
yes.. fixed it